camel/societies/role_playing.py

```diff
@@ -363,13 +362,13 @@ def init_chat(self) -> Tuple[BaseMessage, List[BaseMessage]]:
         """
         self.assistant_agent.reset()
         self.user_agent.reset()
-
+        default_msg = ("Now start to give me instructions one by one. "
+                       "Only reply with Instruction and Input.")
+        if first_msg is None:
+            first_msg = default_msg
         # Send the system messages again to the agents using chat messages
         assistant_msg = BaseMessage.make_assistant_message(
-            role_name=self.assistant_sys_msg.role_name,
-            content=(f"{self.user_sys_msg.content}. "
-                     "Now start to give me instructions one by one. "
-                     "Only reply with Instruction and Input."))
+            role_name=self.assistant_sys_msg.role_name, content=first_msg)
         user_msg = BaseMessage.make_user_message(
```

Comment:
It seems we are still repeating the system message for the assistant agent, right?

Reply:
> It seems we are still repeating the system message for the assistant agent, right?

No, actually only the `content` here matters. `assistant_sys_msg.role_name` is more of a placeholder since during `step` each agent knows how to recode the role of incoming message correctly. I just keep the originally code of this part (well it could be confusing I should say).

Reply:
I meant this PR seems to be not yet solving the consecutive user msg bug.

Reply:
> I meant this PR seems to be not yet solving the consecutive user msg bug.

I'm so sorry about my mistake. It should be fixed now after deleting the `.step`.